src/poetry/exceptions.py

```diff
@@ -7,3 +7,7 @@ class PoetryException(Exception):
 
 class InvalidProjectFile(PoetryException):
     pass
+
+
+class GroupNotFound(PoetryException):
+    pass
```

Comment:
We don't have a clear concept for exceptions, but I think this one fits better in `poetry.console.exceptions`.